packages/calcite-components/src/components/alert/alert.e2e.ts

```diff
@@ -594,4 +594,37 @@ describe("calcite-alert", () => {
   describe("translation support", () => {
     t9n("calcite-alert");
   });
+
+  describe("theme", () => {
+    themed(
```

Comment:
break this into a "default" alert test followed by additional tests to cover tokens only effected by certain props. I.E. Slotted content styles